src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -145,12 +147,22 @@ const StripePaymentForm = ({
         options={options}
         onReady={stripeElementsOnReady}
       />
-      <PlaceOrderButton
-        onSubmitButtonClick={onSubmitButtonClick}
-        showLoadingButton={showLoadingButton}
-        disabled={submitting}
-        isProcessing={isProcessing}
-      />
+      {isSubscription ? (
+        <SubscriptionSubmitButton
+          onSubmitButtonClick={onSubmitButtonClick}
+          showLoadingButton={showLoadingButton}
+          disabled={submitting}
+          isProcessing={isProcessing}
+        />
+      ) : (
+        <PlaceOrderButton
```

Comment:
Can you rename `PlaceOrderButton` component to `SubmitButton`? This will better match the subs button component name plus "place order" is the text in the button which could change in the future, so the name won't be relevant

Reply:
Will cover in the next subsequent PR against this [ticket](https://2u-internal.atlassian.net/browse/PON-82)